back/sssscs/src/main/java/com/ib/util/security/JwtRequestFilter.java

```diff
@@ -28,23 +31,33 @@ public class JwtRequestFilter extends OncePerRequestFilter {
 	private JwtTokenUtil jwtTokenUtil;
 
 	@Override
-	protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain)
-			throws ServletException, IOException {
-		this.doJwtAuthenticationFilter(request);
+	protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws ServletException, IOException {
+		this.doJwtAuthenticationFilter(request, response);
 		chain.doFilter(request, response);
 	}
 
-	private void doJwtAuthenticationFilter(HttpServletRequest request) {
+	private void doJwtAuthenticationFilter(HttpServletRequest request, HttpServletResponse response) {
 		System.err.println(request.getRequestURL().toString() + " " + request.getMethod());
 		if (!request.getRequestURL().toString().contains("/api/")) {
 			return;
 		}
-		String jwtToken = this.getTokenFromRequest(request);
-		var authenticationToken = this.getAuthFromToken(jwtToken);
-		if (authenticationToken == null) {
+
+		try {
+			String jwtToken = this.getTokenFromRequest(request);
+			var authenticationToken = this.getAuthFromToken(jwtToken);
+			if (authenticationToken == null) {
+				return;
```

Comment:
This being null was supposed to indicate an error, but since we now propagate the error in `getAuthFromToken` instead of catching it and returning null, we don't need this check anymore.

Reply:
[99767f5](https://github.com/magley/sssscs/pull/40/commits/99767f51064d36e209868d708a8d3998e426698c)